### DISCUSSION TITLE
[PUG Manaus] Apoio da APyB

### DISCUSSION THREAD

**Discussion by meu-bem** [apyb/comunidade#184] [Pergunta]
Oi, pessoal! Tudo bem?

Recentemente iniciamos o PUG - Manaus (Python Users Goup de Manaus) e vamos fazer no nosso [primeiro evento](https://www.even3.com.br/1-meetup-pug-manaus/) agora em Dezembro, um evento beneficente, inclusive. Eu acabei de solicitar o apoio da APyB ao evento e gostaria de me certificar de que fizemos tudo corretamente e receber a confirmação do lado de vocês.

Muito obrigado, pessoal!

cc @natasharhayane @jhonferreira17 @julianyraiol

**Comment by tiidadavena** [comment]
Confirmamos que o registro foi feito de forma correta. 

Parabéns e sucesso para o PUG - Manaus! 🐍
